shell/src/main/java/tachyon/shell/command/AbstractTfsShellCommand.java

```diff
@@ -51,4 +56,40 @@ public boolean validateArgs(String... args) {
     }
     return valid;
   }
+
+  /**
+   * Gets the expected number of arguments of the command.
+   *
+   * @return the number of arguments
+   */
+  abstract int getNumOfArgs();
+
+  @Override
+  public CommandLine parseAndValidateArgs(String... args) {
+    Options opts = new Options();
```

Comment:
as discussed, we may want to let each subclass of `AbstractTfsShellCommand` to define its own `Options`, but we could still do the parsing in here.